spec/comment_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
```

Comment:
- [ ] I think this should be `rails_helper` else it prevents the tests from running.

  ![image](https://user-images.githubusercontent.com/36542986/221372020-189d5f51-ab17-4247-98b1-41c513f8ebbf.png)

- [ ] **[OPTIONAL]** I think it would be a better idea to move this test into a `spec/models` folder, instead of just leaving it as `spec` as it makes for a better organization.